<?xml version="1.0"?>
<launch>

     <group ns="robot_0">
          <node pkg="explore_lite" type="explore" respawn="false" name="explore0" output="screen">
               <param name="robot_base_frame" value="robot_0/base_link"/>
               <param name="costmap_topic" value="robot_0/map_fused"/>
               <param name="costmap_updates_topic" value="map_updates"/>
               <param name="visualize" value="true"/>
               <param name="planner_frequency" value="0.33"/>
               <param name="progress_timeout" value="30.0"/>
               <param name="potential_scale" value="3.0"/>
               <param name="orientation_scale" value="0.0"/>
               <param name="gain_scale" value="1.0"/>
               <param name="transform_tolerance" value="0.3"/>
               <param name="min_frontier_size" value="0.75"/>
          </node>
     </group>

     <group ns="robot_1">
          <node pkg="explore_lite" type="explore" respawn="false" name="explore1" output="screen">
               <param name="robot_base_frame" value="robot_1/base_footprint"/>
               <param name="costmap_topic" value="robot_1/move_base_node/global_costmap/costmap"/>
               <param name="costmap_updates_topic" value="/robot_1/move_base_node/global_costmap/costmap_updates"/>
               <param name="visualize" value="true"/>
               <param name="planner_frequency" value="0.33"/>
               <param name="progress_timeout" value="30.0"/>
               <param name="potential_scale" value="3.0"/>
               <param name="orientation_scale" value="0.0"/>
               <param name="gain_scale" value="1.0"/>
               <param name="transform_tolerance" value="0.3"/>
               <param name="min_frontier_size" value="0.75"/>
          </node>
     </group>

     <group ns="robot_2">
          <node pkg="explore_lite" type="explore" respawn="false" name="explore2" output="screen">
               <param name="robot_base_frame" value="robot_2/base_link"/>
               <param name="costmap_topic" value="robot_2/map_fused"/>
               <param name="costmap_updates_topic" value="map_updates"/>
               <param name="visualize" value="true"/>
               <param name="planner_frequency" value="0.33"/>
               <param name="progress_timeout" value="30.0"/>
               <param name="potential_scale" value="3.0"/>
               <param name="orientation_scale" value="0.0"/>
               <param name="gain_scale" value="1.0"/>
               <param name="transform_tolerance" value="0.3"/>
               <param name="min_frontier_size" value="0.75"/>
          </node>
     </group>
</launch>